jax/_src/numpy/fft.py

```diff
@@ -444,15 +444,162 @@ def _fft_core_2d(func_name: str, fft_type: xla_client.FftType, a: ArrayLike,
   return _fft_core(func_name, fft_type, a, s, axes, norm)
 
 
-@implements(np.fft.fft2)
 def fft2(a: ArrayLike, s: Shape | None = None, axes: Sequence[int] = (-2,-1),
          norm: str | None = None) -> Array:
+  r"""Compute a two-dimensional discrete Fourier transform along given axes.
```

Comment:
I don't think a raw string is necessary here – I don't see any escapted characters in the docstring.

Reply:
Removed